wd_helper.au3

```diff
@@ -203,7 +204,7 @@ Func _WD_Attach($sSession, $sString, $sMode = Default)
 
 	If $sMode = Default Then $sMode = 'title'
 
-	$aHandles = _WD_Window($sSession, 'handles')
+	$aHandles = _WD_Window($sSession, 'handles') ; This function change $_WD_HTTPRESULT
```

Comment:
I don't think this comment is needed here

Reply:
Only this one or also other locations?

Reply:
Other locations as well 🙂

Reply:
Done